dipy/reconst/fwdti.py

```diff
@@ -729,7 +729,7 @@ def nls_fit_tensor(
     weighting=None,
     sigma=None,
 ):
-    """
+    r"""
```

Comment:
same here

Reply:
My bad. I may have added it thinking that a warning in this method was raised by some math expression, but no backslash is present in this method. Removed in 1b8fea5.